docs/design/README.md

```diff
@@ -477,8 +476,6 @@ may be limited to integers of at most 128 bits due to LLVM limitations.
 >     [#861: Naming conventions](https://github.com/carbon-language/carbon-lang/pull/861)
 > -   Proposal
 >     [#1083: Arithmetic expressions](https://github.com/carbon-language/carbon-lang/pull/1083)
-> -   Proposal
->     [#2015: Numeric type literal syntax](https://github.com/carbon-language/carbon-lang/pull/2015)
```

Comment:
Seems better to keep this -- it still seems relevant to readers of this section who want to learn more.